src/lang/english.txt

```diff
@@ -2362,8 +2362,9 @@ STR_NETWORK_SERVER_LIST_MAP_SIZE_CAPTION                        :{BLACK}Map size
 STR_NETWORK_SERVER_LIST_MAP_SIZE_CAPTION_TOOLTIP                :{BLACK}Map size of the game{}Click to sort by area
 STR_NETWORK_SERVER_LIST_DATE_CAPTION                            :{BLACK}Date
 STR_NETWORK_SERVER_LIST_DATE_CAPTION_TOOLTIP                    :{BLACK}Current date
-STR_NETWORK_SERVER_LIST_YEARS_CAPTION                           :{BLACK}Years
-STR_NETWORK_SERVER_LIST_YEARS_CAPTION_TOOLTIP                   :{BLACK}Number of years{}the game is running
+STR_NETWORK_SERVER_LIST_PLAY_TIME_SHORT                         :{BLACK}{NUM}h {NUM}m
```

Comment:
Given I have the feeling the web translator shows the strings in a more-or-less random order,  will the translators have enough context to determine what `h` and `m` mean? Maybe adding a bit of information to the string name will go a long way. Also holds for the non-`_SHORT` version.
```suggestion
STR_NETWORK_SERVER_LIST_PLAY_TIME_SHORT_HOURS_MINUTES           :{BLACK}{NUM}h {NUM}m
```

Reply:
I was going for consistency, where a bit higher we have `MAP_SIZE_SHORT` doing the same thing. I was thinking that would give translators a good idea that these are around the same place.

In general, we don't actually postfix string-names with the content of the value.

Reply:
To put Discord here:

Ideally, we have something like a `{DURATION}` to resolve this, as that would both de-duplicate the entries and make it much more clear to translators what is going on.

However, the exact syntax is something we need to work out, and possibly it can be used in other places too. For that reason, not for this PR.

With that knowledge, I rather leave the strings as they are, as if we would be to add `{DURATION}`, having a postfix of `_HOURS_MINUTES` would be weird (and renaming is expensive).